### PR TITLE
[FLINK-3567] Fix selection on grouping. Only grouping keys or aggregates allowed.

### PR DESCRIPTION
**Pull request by fhueske**

### CODE REVIEW

File: flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala [hunk 1]
```diff
@@ -385,16 +387,23 @@ class GroupedTable(
...
-      .map(toRexNode(_, relBuilder))
+    val exprs: List[RexNode] = try {
+       extractedAggCalls
+        .map(_._1)
+        .map(toRexNode(_, relBuilder))
+    }
+    catch {
+      case iae: IllegalArgumentException  =>
+        throw new IllegalArgumentException(
+        "Only grouping fields and aggregations allowed after groupBy.", iae)
```
Comment (twalthr) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala]: Wrong indention.

**Comment by twalthr** [comment]
Looks good to merge (besides a small comment).

**Comment by vasia** [comment]
Merging this one, too.

**Comment by vasia** [comment]
You can close this @fhueske!

**Comment by fhueske** [comment]
Thanks!

**Pull request closed without merging**